server/sonar-web/src/main/js/apps/projects-admin/main.js

```diff
@@ -191,12 +195,12 @@ export default class Main extends React.PureComponent {
     );
   };
 
-  onProjectSelected = (project: { key: string }) => {
+  onProjectSelected = (project /*: { key: string } */) => {
     const newSelection = uniq([].concat(this.state.selection, project.key));
     this.setState({ selection: newSelection });
   };
 
-  onProjectDeselected = (project: { key: string }) => {
+  onProjectDeselected = (project /*: { key: string } */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)